pkg/scheduler/core/extender.go

```diff
@@ -84,8 +84,8 @@ func makeTransport(config *schedulerapi.Extender) (http.RoundTripper, error) {
 
 // NewHTTPExtender creates an HTTPExtender object.
 func NewHTTPExtender(config *schedulerapi.Extender) (framework.Extender, error) {
-	if config.HTTPTimeout.Nanoseconds() == 0 {
-		config.HTTPTimeout = time.Duration(DefaultExtenderTimeout)
+	if config.HTTPTimeout.Duration.Nanoseconds() == 0 {
```

Comment:
nit: `.IsZero()`

Reply:
That's a function in `time.Time` not `time.Duration`. Unless you meant something else?

Reply:
You are right, sorry.